code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,340 @@
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_HARM)
+		next_command = world.time + cooldown_meme
+
+	//THROW/CATCH
+	else if(findtext(command, "throw") || findtext(command, "catch"))
+		for(var/mob/living/carbon/C in listeners)
+			C.throw_mode_on()
+		next_command = world.time + cooldown_meme
+
+	//FLIP
+	else if(findtext(command, "flip") || findtext(command, "rotate") || findtext(command, "revolve") || findtext(command, "roll") || findtext(command, "somersault"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("flip")
+		next_command = world.time + cooldown_meme
+
+	//REST
+	else if(findtext(command, "rest"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(!L.resting)
+				L.lay_down()
+		next_command = world.time + cooldown_meme
+
+	//GET UP
+	else if(findtext(command, "get up"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(L.resting)
+				L.lay_down() //aka get up
+			L.SetWeakened(0)
+			L.SetParalysis(0)
+		next_command = world.time + cooldown_damage
+
+	//SIT
+	else if(findtext(command, "sit"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			for(var/obj/structure/chair/chair in get_turf(L))
+				chair.buckle_mob(L)
+				break
+		next_command = world.time + cooldown_meme
+
+	//STAND UP
+	else if(findtext(command, "stand"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(L.buckled && istype(L.buckled, /obj/structure/chair))
+				L.buckled.unbuckle_mob(L)
+		next_command = world.time + cooldown_meme
+
+	//DANCE
+	else if(findtext(command, "dance"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("dance")
+		next_command = world.time + cooldown_meme
+
+	//JUMP
+	else if(findtext(command, "jump"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.say("HOW HIGH?!!")
+			L.emote("jump")
+		next_command = world.time + cooldown_meme
+
+	//SALUTE
+	else if(findtext(command, "salute"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("salute")
+		next_command = world.time + cooldown_meme
+
+	//PLAY DEAD
+	else if(findtext(command, "play dead"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("deathgasp")
+		next_command = world.time + cooldown_meme
+
+	//PLEASE CLAP
+	else if(findtext(command, "clap") || findtext(command, "applaud"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("clap")
+		next_command = world.time + cooldown_meme
+
+	//HONK
+	else if(findtext(command, "honk"))
+		playsound(get_turf(owner), "sound/items/bikehorn.ogg", 300, 1)
+		if(owner.mind && owner.mind.assigned_role == "Clown")
+			for(var/mob/living/carbon/C in listeners)
+				C.slip(0,5)
+		next_command = world.time + cooldown_meme
+
+	//RIGHT ROUND
+	else if(findtext(command, "like a record baby"))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.SpinAnimation(speed = 10, loops = 5)
+		next_command = world.time + cooldown_meme
+
+	else
+		next_command = world.time + cooldown_none
```

Comment:
Instead of repeating
`		next_command = world.time + cooldown_meme
on every possible action, just have that set by default, and set no cooldown on the else.